src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -13,6 +13,11 @@ public partial class ImportExport
         [Fact]
         public static void ExportAutoKey()
         {
+            if (PlatformDetection.IsSmallerThan462)
```

Comment:
We can do that, the problem is that ConditionalFact is not working yet on Desktop. It won't discover the test, I'm investigating that as well, so I can do that but it won't be discovered :(